src/main/proto/pb.proto

```diff
@@ -1335,8 +1335,7 @@ message Tx {
     repeated BaseTxOutput tx_outputs = 1;
     TxType txType = 2;
     int64 burnt_fee = 3;
-    int32 lock_time = 4;
-    int32 unlock_block_height = 5;
+    int32 unlock_block_height = 4;
```

Comment:
This `.proto` change is backwards compatible, but in conversation @ManfredKarrer confirmed that these messages are DAO-related and not yet live, so I believe this should be safe.